Add comment to clarify without_internet usage in an example

ErdaradunGaztea opened the pull request:
Related to issue #67 I made earlier. Added a comment to the examples that clarifies why `without_internet()` was used. I thought that function was added to avoid making internet calls in examples (which was a problem when I co-submitted a certain package to CRAN); the comment should clear any possible confusion from people that skip past the title and the description straight to the examples...

nealrichardson commented on R/expect-request.R:
```diff
@@ -27,6 +27,7 @@
 #' @return A `testthat` 'expectation'.
 #' @examples
 #' library(httr)
+#' # without_internet provides required mock context for expectations
```
Thanks for suggesting this clarification. I think it should be more prominent than this though. What do you think about adding a paragraph/sentence around L11 above that says that these expect_* functions only work in the mock contexts, that they won't do anything if you call them outside?

ErdaradunGaztea replied:
Okay, can do!

ErdaradunGaztea commented:
I added a paragraph, as promised. And, besides, noticed that you used `R.Version()` and then pasted the components together. I replaced this part with much simpler `getRversion()`; from base package as well, so no dependencies were introduced. This is a `numeric_version` object, so a `<` comparison with string will work the same as `utils::compareVersion()`.

nealrichardson reviewed (approved):
Thanks a lot!
